neqo-transport/src/dump.rs

```diff
@@ -9,11 +9,11 @@
 
 use crate::connection::Connection;
 use crate::frame::Frame;
-use crate::packet::PacketHdr;
+use crate::packet::{PacketNumber, PacketType};
 use neqo_common::{qdebug, Decoder};
 
 #[allow(clippy::module_name_repetitions)]
-pub fn dump_packet(conn: &Connection, dir: &str, hdr: &PacketHdr, payload: &[u8]) {
+pub fn dump_packet(conn: &Connection, dir: &str, pn: PacketNumber, pt: PacketType, payload: &[u8]) {
```

Comment:
BuildPacketHeader outputs pt, pn. Suggest dump_packet also take them in this order, to salve my OCD.